actionpack/lib/action_dispatch/http/filter_redirect.rb

```diff
@@ -32,6 +34,24 @@ def location_filter_match?
         end
       end
 
+      DEFAULT_PARAMETER_FILTER = []
+      def parameter_filter
+        ParameterFilter.new request.fetch_header('action_dispatch.parameter_filter') {
+          DEFAULT_PARAMETER_FILTER
+        }
+      end
+
+      KV_RE   = '[^&;=]+'
+      PAIR_RE = %r{(#{KV_RE})=(#{KV_RE})}
```

Comment:
Feel like these two constants could be renamed or expanded for more readability.